game/tests/test_init_game_data.py

```diff
@@ -0,0 +1,16 @@
+import pytest
+from django.core.management import call_command
+from game.models import CardSuit, CardRank, Card
```

Comment:
PEP8, but i guess in this case you can leave like it is:
```suggestion
from django.core.management import call_command

from game.models import CardSuit, CardRank, Card
```